src/utils/graphql/queries/getTemplate.query.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getTemplate($code: String!, $status: TemplateStatus = AVAILABLE) {
+    templates(condition: { code: $code, status: $status }) {
```

Comment:
You used `condition` here, so keeping it all the same except where we need advanced filtering makes it slightly more consistent.